apps/full-site-editing/full-site-editing-plugin/lib/feature-flags/feature-flags.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Feature flags file.
+ *
+ * @package full-site-editing
+ */
+
+/**
+ * Class A8C_Full_Site_Editing_Feature_Flags
+ */
+class A8C_Full_Site_Editing_Feature_Flags {
+	/**
+	 * Class instance.
+	 *
+	 * @var A8C_Full_Site_Editing_Feature_Flags
+	 */
+	private static $instance = null;
+
+	/**
+	 * Feature flags.
+	 *
+	 * @var array
+	 */
+	private $flags = array();
+
+	/**
+	 * A8C_Full_Site_Editing_Feature_Flags constructor.
+	 */
+	private function __construct() {
+		$this->set_flags();
+	}
+
+	/**
+	 * Creates instance.
+	 *
+	 * @return \A8C_Full_Site_Editing_Feature_Flags
+	 */
+	public static function get_instance() {
+		if ( null === self::$instance ) {
```

Comment:
PHP also has a `is_null` function, just FYI

Reply:
Updated in 011120a668d1d0887a6a013658f1c06b012a69d5.

Reply:
😞